extensions.json

```diff
@@ -1307,7 +1307,10 @@
     },
     {
         "id": "erlang-ls.erlang-ls",
-        "repository": "https://github.com/erlang-ls/vscode"
+        "repository": "https://github.com/erlang-ls/vscode",
+        "checkout": "master",
+        "prepublish": "npm run compile",
```

Comment:
I notice that the [vscode:prepublish](https://github.com/erlang-ls/vscode/blob/79b7b5b8a180f00c640d9aa772e2fc4d9d7180e7/package.json#L22) NPM script already runs `cd client && npm run compile`, but I agree that the [compile](https://github.com/erlang-ls/vscode/blob/79b7b5b8a180f00c640d9aa772e2fc4d9d7180e7/package.json#L23) NPM script is more comprehensive as it also runs:

```
cd erlang_ls && rebar3 as debug escriptize
```

So, okay for adding `npm run compile` as an extra `prepublish` step here.

Reply:
Acknowledged

Reply:
Resolved in the latest commit. Please verify.